composer.json

```diff
@@ -76,25 +79,26 @@
         "psr/http-server-handler": "^1.0",
         "psr/http-server-middleware": "^1.0",
         "s9e/text-formatter": "^2.3.6",
-        "symfony/config": "^4.3.4",
-        "symfony/console": "^4.3.4",
-        "symfony/event-dispatcher": "^4.3.4",
+        "symfony/config": "^v5.2.2",
+        "symfony/console": "^5.2.2",
```

Comment:
This should probably also have a v for consistency?

Reply:
Removed all v's because they add no value and no other dep has it.